app/docs/0.13.x/admin-api.md

```diff
@@ -1257,64 +1258,8 @@ Attributes | Description
 ---:| ---
 `id`<br>*optional* | A filter on the list based on the `id` field.
 `name`<br>*optional* | A filter on the list based on the `name` field.
-`api_id`<br>*optional* | A filter on the list based on the `api_id` field.
-`consumer_id`<br>*optional* | A filter on the list based on the `consumer_id` field.
-`size`<br>*optional, default is __100__* | A limit on the number of objects to be returned.
-`offset`<br>*optional* | A cursor used for pagination. `offset` is an object identifier that defines a place in the list.
-
-#### Response
-
-```
-HTTP 200 OK
-```
-
-```json
-{
-    "total": 10,
-    "data": [
-      {
-          "id": "4d924084-1adb-40a5-c042-63b19db421d1",
-          "api_id": "5fd1z584-1adb-40a5-c042-63b19db49x21",
-          "name": "rate-limiting",
-          "config": {
-              "minute": 20,
-              "hour": 500
-          },
-          "enabled": true,
-          "created_at": 1422386534
-      },
-      {
-          "id": "3f924084-1adb-40a5-c042-63b19db421a2",
-          "api_id": "5fd1z584-1adb-40a5-c042-63b19db49x21",
-          "consumer_id": "a3dX2dh2-1adb-40a5-c042-63b19dbx83hF4",
-          "name": "rate-limiting",
-          "config": {
-              "minute": 300,
-              "hour": 20000
-          },
-          "enabled": true,
-          "created_at": 1422386585
-      }
-    ],
-    "next": "http://localhost:8001/plugins?size=2&offset=4d924084-1adb-40a5-c042-63b19db421d1"
-}
-```
```

Comment:
It seems like we should keep this example response but update it?

Reply:
I removed the "List Plugins per API" section. The git diff makes it look as if I had removed the previous one (because the responses are very similar in both cases)